js/src/chrome/settings/chrome.settings.filters.js

```diff
@@ -292,10 +288,8 @@ Wu.Chrome.SettingsContent.Filters = Wu.Chrome.SettingsContent.extend({
 		var meta = this._layer.getPostGISData();
 		var rawsql = meta.sql;
 		var table = meta.table_name;
-		var sql = rawsql.replace(table, 'table').replace('  ', ' ');
-
 		// remove (etc) as sub
-		var sql = this._cleanSQL(sql);
+		var sql = this._cleanSQL(rawsql.replace(table, 'table').replace('  ', ' '));
```

Comment:
This is not the same code anymore, @IgorZiegler.  The `sql` variable is modified twice (although it shouldn't be defined with `var` the second time).

Please be very careful when changing code that you don't change the logic.